performance-storage-service/pss_project/api/serializers/rest/metrics/MicrobenchmarkMetricsSerializer.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework.serializers import Serializer, RegexField, DecimalField, IntegerField
+from pss_project.api.models.rest.metrics.MicrobenchmarkMetrics import MicrobenchmarkMetrics
+
+
+class MicrobenchmarkMetricsSerializer(Serializer):
+    # Fields
+    throughput = DecimalField(max_digits=24, decimal_places=15, coerce_to_string=False)
+    name = RegexField(regex=r'^[^/]+/[^/]+$')
```

Comment:
just not sure, what this regex is capturing?

Reply:
Oh its making sure that there is a '/' in the name but I think I can just send the `suite_name` and `test_name` as individual fields instead of parsing it myself